test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -2383,6 +2384,14 @@ def convert_to_cli(self, attrs, op=None, hostname=None, dflt_conf=True,
             attrs = attrs.items()
 
         for a, v in attrs:
+            # In job name string, use prefix "\" with special charater
+            # to read as an ordinary character on
+            # cray, carysim and shasta platform
```

Comment:
s/carysim/craysim/

Reply:
@mike0042 your mean i should use "or" instead of "and" or something else. because i didn't get "carysim" ans "s"

Reply:
No, it's vim syntax. Please change "carysim" to "craysim". And please add a comma after "craysim" once it's changed. I favor use of the Oxford comma when writing lists.